COLLECTOR_VERSION

```diff
@@ -1 +1 @@
-3.20.x-46-gab14f5b5a0
+3.20.x-48-g5a07a9b608
```

Comment:
Updated collector to a version that uses `perContainerRateLimit` rather than `per_container_rate_limit`